### ISSUE TITLE
[FEATURE] Фичи, которых не хватает в виджете "Выбор инструмента"

### ISSUE THREAD

**Issue by offvine** [alor-broker/Astras-Trading-UI#1544]
### Опишите проблему, которую решает ваша фича

По приоритету:

1. Сортировка по ОБОРОТУ В ДЕНЬГАХ
2. Сортировка по количеству сделок
3. Сортировка по текущему спреду
4. Возможность сортировки "Списков инструментов" (которые создаются в настройках виджета)
5. Ручное изменение ширины колонок
6. Вывод в заголовок виджета названия выбранного списка инструментов
7. Чередование цвета строк (например, белый/светло-серый)
8. Иконки/логотипы инструментов

1-3 про удобный отбор инструментов
4-8 про удобство использования

### Опишите предлагаемое вами решение

Лучшим решением будет добавление этих фич

### Правила

- [X] Я согласен следовать правилам

**Comment by offvine** [comment]
Это здорово. А из перечисленного мной уже есть что-то запланированное?

**Comment by PilgrimViis** [comment]
> Это здорово. А из перечисленного мной уже есть что-то запланированное?

Да. Но конкретно что не могу пока говорить. Мы план работ на следующий этап только планируем
